spec/controllers/accounts_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
just a placeholder for a controller test I need to get working - I can uncomment the code and ad `skip` so it shows up in the tests output.